runtime/j9vm/java11vmi.c

```diff
@@ -1561,16 +1552,9 @@ JVM_SetBootLoaderUnnamedModule(JNIEnv *env, jobject module) {
 		j9object_t modObj = J9_JNI_UNWRAP_REFERENCE(module);
 		J9ClassLoader *systemClassLoader = vm->systemClassLoader;
 		J9Class *instanceClazz = J9OBJECT_CLAZZ(currentThread, modObj);
-		J9Class *moduleClass = NULL;
-		if (J2SE_SHAPE(vm) < J2SE_SHAPE_B165) {
-			moduleClass = vmFuncs->internalFindKnownClass(currentThread,
-				J9VMCONSTANTPOOL_JAVALANGREFLECTMODULE,
-				J9_FINDKNOWNCLASS_FLAG_INITIALIZE);
-		} else {
-			moduleClass = vmFuncs->internalFindKnownClass(currentThread,
+		J9Class *moduleClass = vmFuncs->internalFindKnownClass(currentThread,
```

Comment:
This variable can be moved closer to its reference.

Reply:
Done.